### PR TITLE
SED=/usr/bin/sed shim workaround unnecessary as of Homebrew/brew#10802

### PR DESCRIPTION
**Pull request by liyang**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] ~~Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?~~
- [ ] ~~Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?~~
- [ ] ~~Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?~~

-----

Waiting on Homebrew/brew#10802 to be merged. See also Homebrew/linuxbrew-core#22483.

I'm expecting checks to fail for now with

> Error: Files were found with references to the Homebrew shims directory.

hence this PR is still in draft. Once the above is merged I'll trigger another set of checks.

### CODE REVIEW

**Comment by carlocab** [comment]
Thanks for your PR.

Please use the preferred commit-message style for homebrew/core. We put the name of the formula first in commit-message headings, and make sure to modify only one formula file per commit. Thus, your PR should consist of 21 separate commits.

Refer to the [documentation](https://docs.brew.sh/Formula-Cookbook#commit) for more details.

**Comment by liyang** [comment]
> Please use the preferred commit-message style for homebrew/core. We put the name of the formula first in commit-message headings, and make sure to modify only one formula file per commit. Thus, your PR should consist of 21 separate commits.

Gladly! Too many projects squash commits and I've gotten used to that style despite hating it!

**Comment by liyang** [comment]
> Refer to the [documentation](https://docs.brew.sh/Formula-Cookbook#commit) for more details.

Sorry I skimmed through that too quickly, as I thought was just a long HOWTO on `git`.

@carlocab None of the commit message summaries exceed 50 _characters[¹](https://github.com/Homebrew/homebrew-core/pull/72637/commits/e02a8735b94e3d2776cd00a5ce9710e479c62e92)_ now, package names inclusive. :)

**Comment by carlocab** [comment]
Most of these should probably be merged without bottles. Adding a `do not merge` label to indicate that should be done manually.

**Comment by Rylan12** [comment]
I think we'll need CI to run, though, to verify that the changes don't break anything. So a normal merge is fine just no need for revision bumps. I don't think a new rebuild would be a bad thing, right?

**Comment by carlocab** [comment]
In principle no, but a rebuild will discard some older bottles (i.e. some bottles for High Sierra, etc). No need to discard those bottles for what should effectively be a no-op.

**Comment by Rylan12** [comment]
Oh yeah, you're right. Good call 👍

File: Formula/quilt.rb [hunk 1]
```diff
@@ -28,12 +28,6 @@ def install
       "--prefix=#{prefix}",
       "--without-getopt",
     ]
-    on_macos do
-      args << "--with-sed=#{HOMEBREW_PREFIX}/bin/gsed"
-    end
-    on_linux do
-      args << "--with-sed=#{HOMEBREW_PREFIX}/bin/sed"
-    end
```
Comment (Bo98) [inline Formula/quilt.rb:36]: This one was not to dodge the shim. It's so we use the one `gnu-sed` formula instead.

**Review by Bo98 — CHANGES_REQUESTED** [review]

**Comment by iMichka** [comment]
I think this can be rebased and finished, a new brew release has been made and the sed shim is now gone.

**Comment by carlocab** [comment]
Ping, @liyang?

**Pull request closed without merging**